src/python/pants/option/global_options.py

```diff
@@ -300,42 +300,18 @@ def from_options(
                 )
                 execution_headers = auth_plugin_result.execution_headers
                 store_headers = auth_plugin_result.store_headers
-                overridden_opt_log = (
-                    "Overriding `{}` to instead be {} due to the plugin from "
-                    "`--remote-auth-plugin`."
-                )
-                if (
-                    auth_plugin_result.instance_name is not None
-                    and auth_plugin_result.instance_name != instance_name
-                ):
-                    logger.debug(
-                        overridden_opt_log.format(
-                            f"--remote-instance-name={repr(instance_name)}",
-                            repr(auth_plugin_result.instance_name),
-                        )
-                    )
+                plugin_provided_opt_log = "Setting --remote-{opt} is not needed and will be ignored since it is provided by the auth plugin."
```

Comment:
Also, Pants style nit on how to refer to options. Should it be `[GLOBAL].name_of_option` instead of `--option`?

Reply:
> Does the log identify itself as coming from Toolchain? Maybe say "Toolchain auth plugin"?



> Also, Pants style nit on how to refer to options. Should it be `[GLOBAL].name_of_option` instead of `--option`?

the rest of the log messages in this file (and the previous code) uses the `--option` syntax. but I agree with you that it should use the other syntax. so I updated the new code to use that.

Reply:
@tdyas I will have a follow up PR from here: https://github.com/pantsbuild/pants/compare/main...asherf:pants:seinfeld?expand=1 
once this PR lands (I don't want to open yet so I don't end up consuming CI resources)

Reply:
While it is always nice to have more logic upstream, would it be easier for the auth-plugin to do this logging, since it will have more context on who/why it is doing it?

Reply:
@stuhood I somewhat agree, however as the plugin API works right now, the plugin will have to "parse" those options out of the options values container, which is possible, but can cause issues and increase complexity if  pants want change how those options are named or behave in the future (there will need to be a more back and forth between pants releases and plugin release, which is a PITA) so for now, I think having this logic here makes more sense.

Reply:
Sounds good: thanks.